src/libraries/Microsoft.IO.Redist/src/Microsoft.IO.Redist.csproj

```diff
@@ -6,6 +6,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <ClsCompliant>false</ClsCompliant>
     <Nullable>annotations</Nullable>
+    <IncludePlatformAttributes>true</IncludePlatformAttributes>
```

Comment:
Why doesn't this need to include the attribute on its ref assembly but it does on the srouce?

Reply:
great question! the answer is tricky: this project is including a file (called `File.cs` btw) from a different project (`System.IO.FileSystem`) that is using the platform attribute:

https://github.com/dotnet/runtime/blob/e4df6a94e7ed733b8e92626d559710219d660b77/src/libraries/Microsoft.IO.Redist/src/Microsoft.IO.Redist.csproj#L36-L37

https://github.com/dotnet/runtime/blob/e4df6a94e7ed733b8e92626d559710219d660b77/src/libraries/System.IO.FileSystem/Directory.Build.props#L6

so it needs this attribute as well.

Reply:
So then we don't need to update the ref assembly for this guy to reflect that attribute? I guess not because this project only builds for .NET Framework?

Reply:
hmm let me check the types exposed by this assembly

Reply:
> So then we don't need to update the ref assembly for this guy to reflect that attribute? I guess not because this project only builds for .NET Framework?

This particular project has no `ref` folder: https://github.com/dotnet/runtime/tree/e4df6a94e7ed733b8e92626d559710219d660b77/src/libraries/Microsoft.IO.Redist

I dont know why, but I guess that it means that we dont need it?

Reply:
Ah I see. Yeah, then disregard this conversation.